decidim-debates/spec/commands/decidim/debates/admin/create_debate_spec.rb

```diff
@@ -120,4 +123,23 @@
       end
     end
   end
+
+  context "when everything is ok with attachments" do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed?